inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1311,3 +1310,15 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
 
     return newNode;
 }
+
+bool MKLDNNNode::canBePerformedAsScaleShift() const {
+    bool inputsIsConst = true;
+    for (size_t i = 1; i < getParentEdges().size(); i++) {
+        if (!getParentEdgeAt(i)->getParent()->isConstant() || getParentEdgeAt(i)->getParent()->getType() != Input) {
+            inputsIsConst = false;
+        }
+    }
+    return one_of(getAlgorithm(), EltwiseAdd, EltwiseMultiply, EltwiseSubtract, EltwiseDivide, EltwisePrelu, EltwiseMulAdd) && inputsIsConst &&
```

Comment:
For `EltwiseMulAdd` we should also check third input.

Reply:
When we create ```EltwiseMulAdd``` we checked that second input ```Multiply``` and ```Add``` nodes broadcastable to first input, so I believe check on broadcast is redundant.
Or I missed something?